test/inductor/test_compiled_optimizers.py

```diff
@@ -181,7 +191,7 @@ def tearDown(self):
     test_adam_recompile = make_recompile_test(Adam, lr=0.01)
     test_adamw_recompile = make_recompile_test(AdamW, lr=0.01)
     # Need an impl which does not use python scalars
-    # test_adamax_recompile = make_recompile_test(Adamax, lr=0.01)
+    test_adamax_recompile = make_recompile_test(Adamax, kernel_count=2, lr=0.01)
```

Comment:
Could you add more configs to test similar to the multi_tensor configs in test_optim.py?

Reply:
done